config/index.js

```diff
@@ -9,7 +9,14 @@ module.exports = {
     // Paths
     assetsSubDirectory: 'static',
     assetsPublicPath: '/',
-    proxyTable: {},
+    proxyTable: {
+      '/socket.io': {
+        target: 'ws://localhost:8091',
```

Comment:
does this work if https://github.com/OpenZWave/Zwave2Mqtt/blob/master/config/app.js#L4 is not 8091?

Reply:
also as below, if it can be overridden to listen on **not** `localhost` will this work?

Reply:
Let me work on this.

Reply:
Done - documented how to use `SERVER_HOST` - `SERVER_PORT` - `SERVER_SSL` and other environment variables to affect the proxying.

Reply:
Nice catch @chrisns I completely missed this